cpp/tests/utilities/column_utilities.cu

```diff
@@ -69,5 +118,61 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+struct column_view_printer {
+  template <typename Element, typename std::enable_if_t<is_numeric<Element>()>* = nullptr>
+  void operator()(cudf::column_view const& col, const char *delimiter, std::ostream &ostream) {
```

Comment:
This should use `to_host` https://github.com/rapidsai/cudf/issues/3216, which will also enable you to print string columns.